cmd/oras/push.go

```diff
@@ -101,7 +104,13 @@ func runPush(opts pushOptions) error {
 	copyOptions.PreCopy = display.StatusPrinter("Uploading", opts.Verbose)
 	copyOptions.OnCopySkipped = display.StatusPrinter("Exists   ", opts.Verbose)
 	copyOptions.PostCopy = display.StatusPrinter("Uploaded ", opts.Verbose)
-	desc, err := packManifest(ctx, store, annotations, &opts)
+
+	var desc ocispec.Descriptor
+	if opts.artifactType == "" {
+		desc, err = packOCIManifest(ctx, store, annotations, &opts)
+	} else {
```

Comment:
Since ORAS artifact don't have config, we need to check if `manifestConfigRef` is empty. If not, output a warning tp remind user that the parameter will be ignored.